src/Workspaces/Core/Portable/Workspace/Solution/RecoverableTextAndVersion.cs

```diff
@@ -143,7 +139,7 @@ public async ValueTask<VersionStamp> GetVersionAsync(LoadTextOptions options, Ca
             return recoverableText.Version;
         }
 
-        private sealed class RecoverableText : WeaklyCachedRecoverableValueSource<SourceText>, ITextVersionable
+        private sealed class RecoverableText : WeaklyCachedRecoverableValueSource<SourceText>
```

Comment:
didn't need this interface here.   RecoverableText is never exposed outside of RecoverableTextAndVersion.  So everything in RecoverableTextAndVersion just casts directly to RecoverableText and can grab anything it needs off of that.